rotkehlchen/db/history_events.py

```diff
@@ -236,23 +235,28 @@ def delete_events_by_location(
     ) -> None:
         """Delete all relevant non-customized events for a given location
 
-        Also set evm_tx_mapping as non decoded so they can be redecoded later
+        Also set evm_tx_mapping as non decoded so they can be re-decoded later
```

Comment:
This function and its docstring should be renamed. This is not events. This is only evm events. If it was all events it should be as it was before.